content/10-minute-tutorial.md

```diff
@@ -110,12 +146,102 @@ You should see something like the following:
 
 Your project builds correctly, but nothing is tested yet as you have not specified any behaviour to test against.
 
+{{% /block %}}
+
+{{% block "javascript" %}}
+
+Create a new working directory called `cucumber-js` and change into it. The rest of tutorial assumes that you are in this directory.
+
+We want to add a `package.json` file to our project and add Cucumber to it.
+
+We can have npm create the `package.json` for us. To get a default package.json, run `npm init -y` (or `npm init -yes`).
+
+This will generate a default `package.json` using information extracted from the current directory:
+
+```
+Wrote to /Users/maritvandijk/cucumberjs/package.json:
```

Comment:
If you move this line out of the block, we can use json highlighting for this block.

Reply:
Since the json is only relevant to the javscript block, I'll try marking the json as such.